src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+
+<figure markdown>
+<table>
+  <thead>
+    <tr>
+      <th>Rank</th>
+      <th>`pre`&shy;`load`</th>
+      <th>`pre`&shy;`fetch`</th>
+      <th>`pre`&shy;`connect`</th>
+      <th>`pre`&shy;`render`</th>
+      <th>`dns-prefetch`</th>
+      <th>`module`&shy;`preload`</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="numeric">1,000</td>
+      <td class="numeric">3</td>
+      <td class="numeric">2</td>
+      <td class="numeric">4</td>
+      <td class="numeric">0</td>
+      <td class="numeric">4</td>
+      <td class="numeric">1</td>
+    </tr>
+    <tr>
+      <td class="numeric">10,000</td>
+      <td class="numeric">3</td>
+      <td class="numeric">1</td>
+      <td class="numeric">4</td>
+      <td class="numeric">1</td>
+      <td class="numeric">3</td>
+      <td class="numeric">1</td>
+    </tr>
+    <tr>
+      <td class="numeric">100,000</td>
+      <td class="numeric">2</td>
+      <td class="numeric">2</td>
+      <td class="numeric">3</td>
+      <td class="numeric">1</td>
+      <td class="numeric">3</td>
+      <td class="numeric">1</td>
+    </tr>
+    <tr>
+      <td class="numeric">1,000,000</td>
+      <td class="numeric">2</td>
+      <td class="numeric">2</td>
+      <td class="numeric">2</td>
+      <td class="numeric">1</td>
+      <td class="numeric">2</td>
+      <td class="numeric">1</td>
+    </tr>
+    <tr>
+      <td class="numeric">all</td>
+      <td class="numeric">2</td>
+      <td class="numeric">2</td>
+      <td class="numeric">1</td>
+      <td class="numeric">1</td>
+      <td class="numeric">2</td>
+      <td class="numeric">1</td>
+    </tr>
+  </tbody>
+</table>
+<figcaption>{{ figure_link(caption='Median resource hint adoption by rank.', sheets_gid="528380369", sql_file="resource_hints_distribution_by_rank.sql") }}</figcaption>
+</figure>
+
+Of the sites using resource hints, when comparing the median for the top 1,000 sites to the entire corpus, the top-ranking sites have more resource hints per page. The only hint which observes a different pattern is `prerender`—which has a total of 0 occurrences in the top 1,000 sites.
+
+## Correlation with Core Web Vitals
+
+{{ figure_markup(
+  image="correlation-of-good-cwv-and-preload.png",
+  caption='Correlation between good CWV score and number of `rel="preload"` hints',
+  description='Scatter plot with trend lines showing the number of pages which have a good CWV score and the number of `rel="preload"` hints on that page. 42.8% of desktop pages, and 30.1% of mobile pages with 0 preload hints have a good CWV score. 21.5% of desktop pages and 11.1% of mobiles page, with 20 preload hints have a good CWV score.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=122948841&format=interactive",
```

Comment:
The x-axis label is "Elements per page" which I think is the label used in the sample chart in the data viz gallery. I think we'll need to update the chart and retake the screenshot.